src/test/java/org/rulez/demokracia/pdengine/VoteTest.java

```diff
@@ -20,7 +20,6 @@ public void setUp() throws ReportedException {
 		super.setUp();
 	}
 
-
 	@tested_feature("Vote")
 	@tested_operation("Cast vote")
 	@tested_behaviour("deletes the ballot with ballotId, so only one vote is possible with a ballot")
```

Comment:
This is @tested_behaviour("works only if canVote is true") in #168 
Which is already grabbed by valentinbujdoso.
We will figure out in email what to do with this.